srcpkgs/python3-txtorcon/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'python3-txtorcon'
 pkgname=python3-txtorcon
-version=20.0.0
-revision=2
+version=21.1.0
+revision=1
 wrksrc="txtorcon-${version}"
 build_style=python3-module
 hostmakedepends="python3 python3-setuptools"
 depends="python3-automat python3-Twisted"
+checkdepends="lsof python3-mock python3-Twisted tox"
```

Comment:
* lsof is a runtime dependency: https://github.com/meejah/txtorcon/search?q=lsof
* then `checkdepends="$depends python3-mox tox"` will be more readable
* you can replace tox with python3-pytest